### PR TITLE
fixerMessage for require-jsdoc

### PR DESCRIPTION
**Pull request by schmetzyannick**

### CODE REVIEW

**Comment by schmetzyannick** [comment]
Closes https://github.com/gajus/eslint-plugin-jsdoc/issues/764

File: test/rules/assertions/requireJsdoc.js [hunk 1]
```diff
@@ -3180,6 +3180,91 @@ function quux (foo) {
...
+        return "comment";
+      }
+      `,
+      errors: [
+        {
+          line: 2,
+          message: 'Missing JSDoc comment.',
+        },
+      ],
+      ignoreReadme: true,
```
Comment (brettz9) [inline test/rules/assertions/requireJsdoc.js:3195]: Notwithstanding the fact that we do need more `ignoreReadme` usage for some obscure edge cases, I think we actually want these examples to not be ignored as they demonstrate an option.

```suggestion
```

File: test/rules/assertions/requireJsdoc.js [hunk 2]
```diff
@@ -3180,6 +3180,91 @@ function quux (foo) {
...
```
Comment (brettz9) [inline test/rules/assertions/requireJsdoc.js:3223]: Likewise here...
```suggestion
```

**Review by brettz9 — CHANGES_REQUESTED** [review]
Looks good--just the `ignoreReadme`'s (and rebuilding the docs accordingly).

**Comment by schmetzyannick** [comment]
There was one more ignoreReadme line in the test, I removed them all.

**Comment by brettz9** [comment]
Ok, cool--and if you can run `npm run create-readme` to ensure the output README is updated accordingly...
